specification/baremetalinfrastructure/resource-manager/readme.go.md

```diff
@@ -13,9 +13,20 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2021-08-09
   - tag: package-2020-08-06-preview
 ```
 
+### Tag: package-2021-08-09 and go
+
+These settings apply only when `--tag=package-2021-08-09 --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2021-08-09' && $(go)
+namespace: baremetalinfrastructure
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-08-09/$(namespace)
```

Comment:
We should not put this in the `preview` sub-directory since this is a stable api-version
```suggestion
output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2021-08-09/$(namespace)
```

Reply:
Done, thanks for that catch!